src/utils/distance.js

```diff
@@ -0,0 +1,81 @@
+import dataService from '../services/data-service';
+import Util from './util';
+
+
+const Distance = {
```

Comment:
I  like that you moved logic from `distanceService` to `Distance` util, it does make sense for `Altitude` and `Distance` be in the same place. But don’t stop there, move `getDistance` method to `Distance` util and remove `distanceService` all together.

And if we're removing "meters" as a horizontal unit of measurement, `getDistance` should return distance in km (default distance unit), not meters.

If you need small distances to show up nicely in the UI, it's better to format them in components or in our case in a chart tooltip itself (see my next comment).